### PR TITLE
Feat: add --no-validate option

### PR DESCRIPTION
**Pull request by haroune-mohammedi**

### CODE REVIEW

**Comment by glihm** [comment]
Hello @haroune-mohammedi! Thanks for the contribution!

This PR seems to be done with an older version of Katana. Would you mind adjusting the proposition to the new crate layout?

[Here](https://github.com/dojoengine/dojo/blob/fc24a5a23584a847d2a69523671d4926dd899fe7/crates/katana/executor/src/blockifier/mod.rs#L147) is the link of the validate value, which for now is fixed to `true`.

Feel free to reach if you have questions. :+1:

File: crates/katana/core/src/execution.rs [hunk 1]
```diff
@@ -112,6 +112,8 @@ pub struct TransactionExecutor<'a> {
     transactions: std::vec::IntoIter<Transaction>,
     /// The state the transactions will be executed on.
     state: &'a mut CachedStateWrapper<StateRefDb>,
+    ///
```
Comment (tdelabro) [inline crates/katana/core/src/execution.rs:115]: Missing doc

**Review by tdelabro — CHANGES_REQUESTED** [review]
missing doc

**Comment by kariy** [comment]
Closing this in favour of #1176

**Pull request closed without merging**